homework4/test/test3.py

```diff
@@ -0,0 +1,17 @@
+import io
+import sys
+
+import homework4.task3 as hw3
+import pytest
+
+
+def test_my_precious_logger_out(capsys):
+    hw3.my_precious_logger("hello")
+    captured = capsys.readouterr()
+    assert captured.out == "hello\n"
```

Comment:
'my_precious_logger' should write either to out or err. so, if test case is about printing to out, you should also check that there are no messages in err and vice versa